app/models/direct_message_channel.rb

```diff
@@ -9,7 +9,10 @@ def user_can_access?(user)
   end
 
   def chat_channel_title_for_user(chat_channel, acting_user)
-    users = direct_message_users.map(&:user) - [acting_user]
+    users =
+      (direct_message_users.map(&:user) - [acting_user]).map do |user|
+        user ? user : DeletedChatUser.new
```

Comment:
I _think_ this should work as well?

```suggestion
        user || DeletedChatUser.new
```

Reply:
It should yes 😅 too much js